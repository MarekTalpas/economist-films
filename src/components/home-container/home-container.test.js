// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import HomeContainer from './home-container';

test('HomeContainer renders correctly', () => {
  const tree : string = renderer.create(<HomeContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
