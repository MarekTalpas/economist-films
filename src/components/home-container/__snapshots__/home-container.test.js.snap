// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContainer:  renders correctly with data 1`] = `null`;

exports[`HomeContainer:  renders correctly without data 1`] = `
<div
  className="home-container"
>
  <div>
    Slider
  </div>
</div>
`;
